pages/orgs/new.tsx

```diff
@@ -24,18 +25,22 @@ export default function NewOrgPage() {
 
     try {
       const org = await createOrg(orgName);
-      
+
       toast({
         title: "Success!",
-        description: 'New organisation created.',
+        description: "New organisation created.",
         status: "success",
         isClosable: true,
         duration: 6000,
       });
 
       resetForm();
 
-      navigateToUpgradePage(org.orgId);
+      if (parseBooleanEnvValue(window.__env.NEXT_PUBLIC_STRIPE_ENABLED)) {
```

Comment:
Why do you access the window.__env directly instead of using loadConfig which uses it to configure the configuration values?